html5lib/__init__.py

```diff
@@ -19,7 +28,8 @@
 from .serializer import serialize
 
 __all__ = ["HTMLParser", "parse", "parseFragment", "getTreeBuilder",
-           "getTreeWalker", "serialize"]
+           "getTreeWalker", "serialize", "__version__"]
```

Comment:
I don't think we want `__version__` to get imported when importing `*`.

Reply:
As `__version__` is part of the public interface of this module, which `__all__` defines, I think it should be here.

Reply:
I wouldn't think about `__all__` that way. It's really for specifying what gets exported if the user does `from html5lib import *`. I don't want `__version__` to get exported in that scenario. Please undo this change.